Automattic/jetpack
# Jetpack AI Interstitial: fix benefit capitalization

## Summary

In the Jetpack AI interstitial there is a tiny mistake with the capitalization of the last benefit.

Currently it reads:

- Seamless WordPress editor Integration

This PR changes that line to:

- Seamless WordPress editor integration


## Testing instructions:

* Go to My Jetpack.
* Click on the Jetpack AI CTA.
* Ensure you see the correct copy in the following interstitial page.


## Does this pull request change what data or activity we track or use?

No.
